staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -767,6 +769,12 @@ type KubeletConfiguration struct {
 	// Default: "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
 	// +optional
 	VolumePluginDir string `json:"volumePluginDir,omitempty"`
+	// seccompProfileRoot is the directory path for seccomp profiles. <Warning: Alpha feature>
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that changing
+	// it may cause Seccomp not work as expected if an invalid path is set.
+	// Default: "/var/lib/kubelet/seccomp"
+	// +optional
+	SeccompProfileRoot string `json:"seccompProfileRoot,omitempty"`
```

Comment:
It sounds similar to the VolumePluginDir use case. Any reason why the use of `SeccompProfileRoot` over `SeccompProfileDir`?

Reply:
This flag was introduced at https://github.com/kubernetes/kubernetes/pull/25324, renaming can be made in another PR.

Reply:
Thanks for clarifying, I would concur with your assessment.